public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -128,42 +118,33 @@ export class ApiKeysPageUnconnected extends PureComponent<Props, State> {
       );
     }
 
+    const showTable = apiKeysCount > 0;
     return (
       <Page {...defaultPageProps}>
         <Page.Contents isLoading={false}>
-          <ApiKeysController>
-            {({}) => {
-              const showTable = apiKeysCount > 0;
-              return (
-                <>
-                  {apiKeysCount !== 0 && <MigrateToServiceAccountsCard onMigrate={this.onMigrateAll} />}
-                  {apiKeysCount === 0 && (
-                    <APIKeysMigratedCard onHideApiKeys={this.onHideApiKeys} apikeys={apiKeysCount} />
-                  )}
-                  {showTable ? (
-                    <ApiKeysActionBar
-                      searchQuery={searchQuery}
-                      disabled={!canCreate}
-                      onSearchChange={this.onSearchQueryChange}
-                    />
-                  ) : null}
-                  {showTable ? (
-                    <VerticalGroup>
-                      <InlineField disabled={includeExpiredDisabled} label="Include expired keys">
-                        <InlineSwitch id="showExpired" value={includeExpired} onChange={this.onIncludeExpiredChange} />
-                      </InlineField>
-                      <ApiKeysTable
-                        apiKeys={apiKeys}
-                        timeZone={timeZone}
-                        onMigrate={this.onMigrateApiKey}
-                        onDelete={this.onDeleteApiKey}
-                      />
-                    </VerticalGroup>
-                  ) : null}
-                </>
-              );
-            }}
-          </ApiKeysController>
+          <>
```

Comment:
I'm not 100% sure, but this container maybe not needed here (if <Page.Contents /> allow multiple elements inside).

Reply:
i see. il try removing it